guiUtils/guiUtils.js

```diff
@@ -0,0 +1,26 @@
+function filterByTestName(search, testName) {
+	if (testName) {
+		if (search) {
+			if (!search.match(/.*testNameFilter=.*/)) {
```

Comment:
No need for wrapping within `.*` in a `match`: you're matching a subset by default.